BlobStorage/GettingStarted.cs

```diff
@@ -66,66 +69,68 @@ private static async Task BasicStorageBlockBlobOperationsAsync()
                 Console.ReadLine();
                 throw;
             }
 
             // To view the uploaded blob in a browser, you have two options. The first option is to use a Shared Access Signature (SAS) token to delegate 
             // access to the resource. See the documentation links at the top for more information on SAS. The second approach is to set permissions 
             // to allow public access to blobs in this container. Uncomment the line below to use this approach. Then you can view the image 
-            // using: https://[InsertYourStorageAccountNameHere].blob.core.windows.net/democontainer/HelloWorld.png
+            // using: https://[InsertYourblobServiceClientNameHere].blob.core.windows.net/democontainer/HelloWorld.png
             // await container.SetPermissionsAsync(new BlobContainerPermissions { PublicAccess = BlobContainerPublicAccessType.Blob });
 
             // Upload a BlockBlob to the newly created container
             Console.WriteLine("2. Uploading BlockBlob");
-            CloudBlockBlob blockBlob = container.GetBlockBlobReference(ImageToUpload);
+            BlockBlobClient blockBlob = container.GetBlockBlobClient(ImageToUpload);
+
+
 
             // Set the blob's content type so that the browser knows to treat it as an image.
-            blockBlob.Properties.ContentType = "image/png";
-            await blockBlob.UploadFromFileAsync(ImageToUpload);
+            await blockBlob.UploadAsync(File.OpenRead(ImageToUpload));
 
             // List all the blobs in the container.
             /// Note that the ListBlobs method is called synchronously, for the purposes of the sample. However, in a real-world
             /// application using the async/await pattern, best practices recommend using asynchronous methods consistently.
             Console.WriteLine("3. List Blobs in Container");
-            foreach (IListBlobItem blob in container.ListBlobs())
+            foreach (var blob in container.GetBlobs())
             {
-                // Blob type will be CloudBlockBlob, CloudPageBlob or CloudBlobDirectory
+                // Blob type will be BlobClient, CloudPageBlob or BlobClientDirectory
                 // Use blob.GetType() and cast to appropriate type to gain access to properties specific to each type
-                Console.WriteLine("- {0} (type: {1})", blob.Uri, blob.GetType());
+                Console.WriteLine("- {0} (type: {1})", blob.Name, blob.GetType());
             }
 
             // Download a blob to your file system
             Console.WriteLine("4. Download Blob from {0}", blockBlob.Uri.AbsoluteUri);
-            await blockBlob.DownloadToFileAsync(string.Format("./CopyOf{0}", ImageToUpload), FileMode.Create);
+            using (FileStream file = File.OpenWrite(string.Format("./CopyOf{0}", ImageToUpload)))
+            {
+                await blockBlob.DownloadToAsync(file);
+            }
```

Comment:
`DownloadTo` takes a file path string in an overload. You could use that unless you have a reason to manage the filestream yourself.